src/main/java/Car.java

```diff
@@ -0,0 +1,41 @@
+import javax.xml.stream.Location;
+
+public class Car {
+    int currentLocation;
```

Comment:
```suggestion
    private int currentLocation;
```
인스턴스 변수는 특별한 이유가 없는 한 private으로 구현